Source/WebCore/html/parser/HTMLEntityTable.h

```diff
@@ -45,6 +47,8 @@ class HTMLEntityTable {
 
     static const HTMLEntityTableEntry* firstEntryStartingWith(UChar);
     static const HTMLEntityTableEntry* lastEntryStartingWith(UChar);
+
+    static const LChar* characters(const HTMLEntityTableEntry&);
```

Comment:
We can refine further later, and this need not be done now. But I meant this should be a `characters()` non-static member function of `HTMLEntityTableEntry`, which should be possible.

Reply:
Created follow-up - https://bugs.webkit.org/show_bug.cgi?id=253360 and if you want me to rename the function to 'entityString', happy to do so. Thanks!

Reply:
I think `characters` is a good name. Should not name it with the word string because it does not return a string.